mq/plugins/stackdriver_audit.py

```diff
@@ -0,0 +1,102 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+
+from mozdef_util.utilities.toUTC import toUTC
+from platform import node
+import os
+import yaml
+import jmespath
+
+# activity for Admin Activity audit logs
+# data_access for Data Access audit logs
+# system_events for System Event audit logs.
+# https://cloud.google.com/logging/docs/api/v2/resource-list
+
+# organization
+# project
+# timestamp
+# authenticationInfo
+# logName
+# protoPayload.serviceName <- what service wrote the audit log
+# protoPayload.methodName <- what operation is being audited
+# protoPayload.serviceData <- more information about the operation being audited
+# resource.type <- What resource is being audited
+# resource.labels.project_id <- What resource is being audited
+# details.gaudit.status.code
+# details.gaudit.status.message
+
+# GCE activity logs
+# set category if jsonPayload in details and logname ends with activity
+# move details.jsonPayload somewhere else
+# details.jsonPayload.actor.user
+# details.jsonPayload.event_subtype
+# details.jsonPayload.event_type
+# details.jsonPayload.ip_address
+# details.logName
+
+# XXX: should I implement a lookup table to map
+
+
+class message(object):
+    def __init__(self):
+        """
+            Plugin used to fix object type discretions with cloudtrail messages
+        """
+        self.registration = ["stackdriver"]
+        self.priority = 15
+
+        try:
+            self.mozdefhostname = "{0}".format(node())
```

Comment:
Similar comment as before, can we delete this block since the variable doesn't appear to be used anywhere?